doc/content/concepts/fetchers.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Fetchers"
+description: ""
+weight: 20
+---
+
+## What is it?
+
+Fetchers allow the stack to gain access to external files, such as [Email Templates]({{< ref "/concepts/email-templates.md" >}}), Frequency Plans, Device Data for the Device Repository, Webhook Templates and others.
+
+## Who is it for?
+
+The fetchers are used by users that want to customize certain parts of the stack. They are exposed indirectly through the stack configuration.
+
+### Typical use cases
+
+1. Providing custom resources to the stack, such as custom [Email Templates]({{< ref "/concepts/email-templates.md" >}}) in order to change the behaviour of the stack.
+2. Allowing the stack to run in air-gapped environments, where internet connections are not available constantly and resources must be served locally.
+
+## How does it work?
+
+The fetchers provide the stack with a way to access files which can be hosted remotely (such as on a remote website), locally (by providing a local path to the files) or in memory (by providing the contents of the files directly as part of the configuration).
+
+### Fetching from a remote URL
+
+In order to allow the stack to access remote files, you must only provide the URL to the root folder that contains the files. Consider that you need to provide the stack access to a file called `temporary_password.txt`, which you have uploaded on your host at `http://www.example.org/emails/temporary_password.txt`. Then the URL that you provide to the email templates configuration is `http://www.example.org/emails/`.
```

Comment:
This would belong in a Guide imo; it's quite specific to what's being fetched, i.e. in this case "the email templates configuration" which is not related to the concept Fetchers, explained here